CIME/Tools/Makefile

```diff
@@ -967,7 +967,7 @@ $(COMPLIB): $(OBJS)
 	$(CXX) -c $(INCLDIR) $(INCS) $(CXXFLAGS)  $<
 
 %.F90: %.F90.in
-	$(CIMEROOT)/src/externals/genf90/genf90.pl $< > $@
+	$(CIMEROOT)/CIME/data/genf90/genf90.pl $< > $@
```

Comment:
why is this here?

Reply:
Is genf90 no longer used or supplied externally. I believe I found references in E3SM so I kept it. @jgfouca do we still used this?

Reply:
genf90 is both used and supplied externally https://github.com/PARALLELIO/genf90

Reply:
@jedwards4b , @jasonb5 , genf90 is used frequently in our build system. I'm wondering if we should bring it in as a submodule instead of having a copy directly in our CIME repo?

Reply:
I agree - it should be a submodule.

Reply:
@jgfouca I can add it as a submodule.

Reply:
So genf90 is no longer needed by cime itself?

Reply:
That's correct, all of the F90 code has been removed from cime.

Reply:
@rljacob , it is still used. I see it being used in the key makefile `scripts/Tools/Makefile` . It also looks like some sharedlibs and tools use it (cprnc and pio).